lib/reversetunnel/fake.go

```diff
@@ -51,14 +51,28 @@ type FakeRemoteSite struct {
 	RemoteSite
 	// Name is the remote site name.
 	Name string
-	// ConnCh receives the connection when dialing this site.
-	ConnCh chan net.Conn
+	// connCh receives the connection when dialing this site.
+	connCh chan net.Conn
+	// closed is set to true after the site is being closed.
+	closed bool
+	// closedMtx is a mutex that protects closed.
+	closedMtx sync.Mutex
```

Comment:
Not sure. I assume it is because we read code top-down and exported vars are usually at the top. I don't remember where I picked this up. [Effective go mentions something similar.](https://go.dev/doc/effective_go#:~:text=Grouping%20can%20also,uint32%0A%20%20%20%20errorCount%20%20uint32%0A)